csvw/csvw.go

```diff
@@ -292,39 +291,27 @@ func newObservationColumn(title, name string) Column {
 // TBC what the final content of these columns should be. For now is a rough port and amalgamation
 // of the code and label columns from the CMD implementation. Will be updated when we have full
 // metadata spec.
-func newLabelColumn(i int, apiDomain string, header []string, dims []dataset.VersionDimension) (Column, error) {
-	dimHeader := header[i]
-	dimHeader = strings.ToLower(dimHeader)
-
-	var dim dataset.VersionDimension
-
-	for _, d := range dims {
-		if d.Name == dimHeader {
-			dim = d
-			break
-		}
-	}
-
-	dimURL := dim.URL
+func newLabelColumn(i int, apiDomain string, header []string, dims dataset.VersionDimension) (Column, error) {
```

Comment:
Just spotted this, now `dims` is no longer a slice it doesn't make so much sense, should probably be called `dim` now.

Reply:
Yeah good point, updated